src/test/java/com/jcabi/http/wire/LastModifiedCachingWireTest.java

```diff
@@ -56,6 +59,9 @@
 
     /**
      * Test body.
+     * @todo: #120 move constants in this file to their tests because tests
```

Comment:
@mkordas So I have to estimate the time to raise a puzzle?! Why didn't mention this earlier the first time you commented on the duplicated puzzle? The Travis thing sounds like a completely unrelated issue to this puzzles actual task.